### PR TITLE
feat: Add ID translation for run stateless application deployment

### PR DESCRIPTION
**Pull request by irvifa**
Related to https://github.com/kubernetes/website/issues/22296
/assign @wahyuoi 
/assign @girikuncoro

### CODE REVIEW

File: content/id/docs/tasks/run-application/_index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Menjalankan "
```
Comment (sftim) [inline content/id/docs/tasks/run-application/_index.md:2]:
```suggestion
title: "Menjalankan"
```

File: content/id/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        nginx-deployment-148880595-fxcez   1/1       Running   0          2m
+        nginx-deployment-148880595-rwovn   1/1       Running   0          2m
+
+## Menghapus Deployment
+
+Menghapus Deployment dengan nama:
+
+    kubectl delete deployment nginx-deployment
+
+## Cara Lama Menggunakan: ReplicationControllers
```
Comment (sftim) [inline content/id/docs/tasks/run-application/run-stateless-application-deployment.md:144]:
```suggestion
## Cara Lama Menggunakan: ReplicationController
```

File: content/id/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    dimana `<nama-pod>` merupakan nama dari Pod kamu.
+
+## Mengubah Deployment
+
+Kamu dapat mengubah Deployment dengan cara mengaplikasikan berkas YAML yang baru.
+Berkas YAML ini memberikan spesifikasi Deployment untuk menggunakan Nginx versi 1.16.1.
+
+{{< codenew file="application/deployment-update.yaml" >}}
+
+1. Aplikasikan berkas YAML yang baru:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/run-application/run-stateless-application-deployment.md:106]: nit!
```suggestion
1. Terapkan berkas YAML yang baru:
```

File: content/id/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+         kubectl apply -f https://k8s.io/examples/application/deployment-update.yaml
+
+2. Perhatikan bahwa Deployment membuat Pod-Pod dengan nama baru dan menghapus Pod-Pod lama:
+
+         kubectl get pods -l app=nginx
+
+## Meningkatkan Jumlah Aplikasi dengan Menginkatkan Ukuran Replika
```
Comment (ariscahyadi) [inline content/id/docs/tasks/run-application/run-stateless-application-deployment.md:114]: typo?
```suggestion
## Meningkatkan Jumlah Aplikasi dengan Meningkatkan Ukuran Replika
```

File: content/id/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+Kamu dapat meningkatkan jumlah Pod di dalam Deployment dengan mengaplikasikan
```
Comment (ariscahyadi) [inline content/id/docs/tasks/run-application/run-stateless-application-deployment.md:116]: nit!
```suggestion
Kamu dapat meningkatkan jumlah Pod di dalam Deployment dengan menerapkan
```

File: content/id/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+berkas YAML baru. Berkas YAML ini akan meningkatkan jumlah replika menjadi 4,
+yang nantinya memberikan spesifikasi agar Deployment memiliki 4 buah Pod.
+
+{{< codenew file="application/deployment-scale.yaml" >}}
+
+1. Aplikasikan berkas YAML:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/run-application/run-stateless-application-deployment.md:122]: nit!
```suggestion
1. Terapkan berkas YAML:
```

File: content/id/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+        kubectl apply -f https://k8s.io/examples/application/deployment-scale.yaml
+
+2. Verifikasi Deployment yang kamu saat ini mmeiliki empat Pod:
```
Comment (ariscahyadi) [inline content/id/docs/tasks/run-application/run-stateless-application-deployment.md:126]: typo?
```suggestion
2. Verifikasi Deployment kamu saat ini yang memiliki empat Pod:
```

File: content/id/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+
+Menghapus Deployment dengan nama:
+
+    kubectl delete deployment nginx-deployment
+
+## Cara Lama Menggunakan: ReplicationController
+
+Cara yang dianjurkan untuk membuat aplikasi dengan replika adalah dengan menggunakan Deployment,
+yang nantinya akan menggunakan ReplicaSet. Sebelum Deployment dan ReplicaSet ditambahkan
+ke Kubernetes, aplikasi dengan replika dikonfigurasi menggunakan [ReplicationController](/docs/concepts/workloads/controllers/replicationcontroller/).
```
Comment (ariscahyadi) [inline content/id/docs/tasks/run-application/run-stateless-application-deployment.md:148]: Link to ID Doc is available.
```suggestion
ke Kubernetes, aplikasi dengan replika dikonfigurasi menggunakan [ReplicationController](/id/docs/concepts/workloads/controllers/replicationcontroller/).
```

File: content/id/docs/tasks/run-application/run-stateless-application-deployment.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+
+
+
+
+## {{% heading "whatsnext" %}}
+
+
+* Pelajari lebih lanjut mengenai [objek Deployment](/docs/concepts/workloads/controllers/deployment/).
```
Comment (ariscahyadi) [inline content/id/docs/tasks/run-application/run-stateless-application-deployment.md:156]: Link to ID Doc is available.
```suggestion
* Pelajari lebih lanjut mengenai [objek Deployment](/id/docs/concepts/workloads/controllers/deployment/).
```

File: content/id/examples/application/deployment-scale.yaml [hunk 10]
```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```
Comment (ariscahyadi) [inline content/id/examples/application/deployment-scale.yaml:1]: We can translate the description also.
```suggestion
apiVersion: apps/v1 # untuk versi sebelum 1.9.0 gunakan apps/v1beta2
```

File: content/id/examples/application/deployment-scale.yaml [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+kind: Deployment
+metadata:
+  name: nginx-deployment
+spec:
+  selector:
+    matchLabels:
+      app: nginx
+  replicas: 4 # Update the replicas from 2 to 4
```
Comment (ariscahyadi) [inline content/id/examples/application/deployment-scale.yaml:9]: We can translate the description also.
```suggestion
  replicas: 4 # Memperbarui replica dari 2 menjadi 4
```

File: content/id/examples/application/deployment-update.yaml [hunk 12]
```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```
Comment (ariscahyadi) [inline content/id/examples/application/deployment-update.yaml:1]:
```suggestion
apiVersion: apps/v1 # untuk versi sebelum 1.9.0 gunakan apps/v1beta2
```

File: content/id/examples/application/deployment-update.yaml [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+      app: nginx
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.16.1 # Update the version of nginx from 1.14.2 to 1.16.1
```
Comment (ariscahyadi) [inline content/id/examples/application/deployment-update.yaml:17]:
```suggestion
        image: nginx:1.16.1 # Memperbarui versi nginx dari 1.14.2 ke 1.16.1
```

**Comment by girikuncoro** [comment]
well translated, thanks for localizing this page @irvifa 

/lgtm
/approve

**Pull request merged**